CODEOWNERS

```diff
@@ -1 +1 @@
-* @jayrbolton
+* @jayrbolton @MrCreosote
```

Comment:
i don't really consider myself an owner - this is your baby. I'm a contributor

Reply:
Ok, removed it.